wallet/wallet.go

```diff
@@ -47,13 +48,20 @@ type (
 	// A Transaction is a transaction relevant to a particular wallet, paired
 	// with useful metadata.
 	Transaction struct {
```

Comment:
Should we consider copying `walletd` and calling these "events"?

Reply:
I am in favor of that. Feels weird to call a miner payout a "transaction."

Reply:
Sounds good to me